src/pages/en/guides/cms/sanity.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Sanity & Astro
+description: Add content to your Astro project using Sanity as a CMS
+layout: ~/layouts/CMSLayout.astro
+stub: true
+---
+
+[Sanity](http://sanity.io) is a headless content management system that focuses on [structured content](https://www.sanity.io/structured-content-platform).
+
+## Community Resources 
+
+- Use the unofficial [`astro-sanity` integration](https://github.com/littlesticks/astro-sanity) to add Sanity to your project.
```

Comment:
Here you mention that this is unofficial, but the netlify-cms one doesn't have this caveat. Do we want to standardize these? Do we want to use our "community" and "official" badges?

Reply:
I think we already cover community vs official in the headings, so probably no need to double up with badges. But might make sense to include “unofficial” on the Netlify CMS integration too.